branch.config

```diff
@@ -10,4 +10,4 @@
 # UPSTREAM_BRANCH parameter should be updated by the release lead on branching
 #
 
-UPSTREAM_BRANCH="master"
+UPSTREAM_BRANCH="6.0/stage"
```

Comment:
Hm.. we'll have to think about how to handle this diff.. might need to do the first merge manually, and tweak the repository configuration to allow this.. I think we need this to remain set to `master`, on the `master` branch..